Gemfile

```diff
@@ -61,6 +61,7 @@ group :development, :test do
   gem "ruby-debug-passenger"
   gem "selenium-webdriver"
   gem "jettywrapper"
+  gem "fcrepo_wrapper"
```

Comment:
These rake tasks never need to be ran in production. It's only rake tasks/cli, I believe. I'll make the change.